app/web/src/features/auth/useAuthStore.ts

```diff
@@ -72,7 +71,7 @@ export default function useAuthStore() {
               action: "logout",
             },
           });
-          setError(isGrpcError(e) ? e.message : ERROR_INFO_FATAL);
+          setError(isGrpcError(e) ? e.message : "");
```

Comment:
What's going on here?

Reply:
I'm hoping I can revert this... it was using the `t` function from `useTranslation` before, but for some strange reasons, a lot of tests are failing when it's **used here only** and not at the rest of the UI, since it just gets stuck at the Suspense fallback state for some reason. Not really sure why  🤷